app/containers/App.js

```diff
@@ -39,6 +39,7 @@ const syncActions = actions.sync;
 import * as actionSources from '../constants/actionSources';
 import { pages, urls, pagesMap } from '../constants/otherConstants';
 import { checkVersion } from '../utils/drivers';
+import DebugMode from '../utils/debugMode';
```

Comment:
If I'm super nit-picky, I would not capitalize `debugMode` to be consistent. But I don't have strong feelings about it.

Reply:
You're right. It doesn't require the `new` keyword, so it probably shouldn't be initial cap.